cfgov/wellbeing/jinja2/wellbeing/results.html

```diff
@@ -230,36 +230,32 @@
 
 
 {% block content_main %}
-    <h1>Your financial well-being score</h1>
+    <h1>{{ _('Your financial well-being score') }}</h1>
+
 {% if user_score %}
     <p class="lead-paragraph">
-        You’ve taken a good step
-        in understanding your financial picture.
+        {{ _('You’ve taken a good step in understanding your financial picture.') }}
     </p>
 {% else %}
     <div class="u-mb15">
         {{ notification.render(
             'warning',
             true,
-            'We never collect or store answers to the financial well-being questionnaire.',
-            'If you would like to see your score and answers on the page below,
-            please start with the <a href="../">questionnaire</a> and enter your responses.' | safe
+            _('We never collect or store answers to the financial well-being questionnaire.'),
+	          _('If you would like to see your score and answers on the page below, please start with the <a href="../">questionnaire</a> and enter your responses.') | safe
         ) }}
     </div>
 {% endif %}
```

Comment:
This notification that appears if you manage to go to the results page without having submitted the form is not being translated to Spanish.

Reply:
Just saw this yesterday, didn't get a chance to fix it yet. If it's not being picked up by `makemessages`, I'll probably take the same approach I used (with Will's blessing) for the text contained inside the views.

Reply:
Addressed in newest commits.